README.md

```diff
@@ -1,73 +1,26 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Awesome Books - ES6
 
-# Project Name
+Web app that allows to add and remove book's data using Javascript and ES6 conventions.
 
-> Description the project.
-
-
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+### Built With
+- HTML
+- CSS
+- Javascript ES6
 
+## Prerequisites
+- Modern web browser with Javascript enabled
+- Install package Luxon
 
+## Install
+git clone git@github.com:rominana/awesome-books-es6.git
 
 ## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
-
-Give a ⭐️ if you like this project!
+- GitHub: @rominana
+- Twitter: @romina_pati
+- Linkedin: romina-patino
 
 ## Acknowledgments
+@microverse
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
+## memo License
+This project is MIT licensed.
```

Comment:
- [ ] Let's make sure that MIT is a link and it should be clickable. Kindly do the same with your LinkedIn, Twitter, and GitHub links. This will make your README file professional.